### PR TITLE
DAOS-7171 script: Fix shellcheck in rpm build.sh

### PR DESCRIPTION
**Pull request by sylviachanoiyee**
Fix shellcheck in rpm build.sh

### CODE REVIEW

File: ci/rpm/build.sh [hunk 1]
```diff
@@ -15,7 +15,7 @@ set -ex
 mydir="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
 ci_envs="$mydir/../parse_ci_envs.sh"
 if [ -e "${ci_envs}" ]; then
-  # shellcheck disable=SC1091
+  # shellcheck disable=SC1091,SC1090
```
Comment (jolivier23) [inline ci/rpm/build.sh:18]: We might need a subsequent enable= to avoid ignoring it for the rest of the file.  Have you checked this?

**Review by jolivier23 — CHANGES_REQUESTED** [review]

File: ci/rpm/build.sh (hunk 1 above)
Comment (sylviachanoiyee) [reply ci/rpm/build.sh:18]: Running a quick test locally like this:
# shellcheck disable=SC1091,SC1090,SC2086
source ${ci_envs}
source ${ci_envs}

shellcheck seems to complain about the second one, not the first. Looks like it only disable the check for the immediate cmd after.

Comment (sylviachanoiyee) [reply ci/rpm/build.sh:18]: yikes, not sure why it end up with crazy formatting above

Comment (JohnMalmberg) [reply ci/rpm/build.sh:18]: Shellcheck ignores generally only apply to the command just following them.
The disable needed for a source command varies with the version of shellcheck, which is why we need to disable both codes.

**Review by JohnMalmberg — APPROVED** [review]

**Review by jolivier23 — APPROVED** [review]

File: ci/rpm/build.sh (hunk 1 above)
Comment (jolivier23) [reply ci/rpm/build.sh:18]: thanks for checking.  That differs from pylint

**Comment by sylviachanoiyee** [comment]
SnapShot Aggregation test failure is known DAOS-7178 and has been added to skip list. Landing this PR.

**Pull request merged**